eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -1063,7 +1063,7 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
         case Failure(cause) => handleLocalError(cause, d, Some(fee))
       }
 
-    case Event(c@CMD_SIGN, d: DATA_SHUTDOWN) =>
+    case Event(c: Command, d: DATA_SHUTDOWN) if c == CMD_SIGN =>
```

Comment:
I think we can do `case Event(c: CMD_SIGN.type, d: DATA_SHUTDOWN)` even in Scala 2.11, why not apply this commit to this PR?

Reply:
Fixed with 53a10ff.